### PR TITLE
refactor(action): compile action from current checkout

### PR DESCRIPTION
**Pull request by chingor13**
You should use the code-suggester library at the tagged version of the action rather than whatever is the latest stable version on npm.

### CODE REVIEW

**Comment by TomKristie** [comment]
How I designed it was to use the latest code-suggester released on npm. I'm just curious about why we want latest master even if we haven't released it. Will that lead to more unexpected behaviour to some users?

**Comment by chingor13** [comment]
It's not using master, it's using whatever is in the repo at that tag. So if you are using the github action at 1.1.1, you should be getting the code-suggester released at 1.1.1, not 1.4 or even 2.0

File: action.yaml [hunk 1]
```diff
@@ -54,15 +54,3 @@ inputs:
 runs:
   using: docker
   image: Dockerfile
-  args:
-    - ${{ inputs.command }}
-    - ${{ inputs.upstream_repo }}
-    - ${{ inputs.upstream_owner }}
```
Comment (bcoe) [inline action.yaml:60]: a little curious about why we can drop this?

Comment (chingor13) [reply action.yaml:60]: GitHub actions provides action inputs as environment variables `INPUT_<underscored name>`. The `entrypoint.sh` script is already using those environment variables, not positional arguments.

**Review by bcoe — APPROVED** [review]

**Pull request merged**